packages/react/src/composite/item/CompositeItem.tsx

```diff
@@ -3,27 +3,46 @@ import * as React from 'react';
 import { useRenderElement } from '../../utils/useRenderElement';
 import { useCompositeItem } from './useCompositeItem';
 import type { BaseUIComponentProps } from '../../utils/types';
+import { EMPTY_OBJECT, EMPTY_ARRAY } from '../../utils/constants';
+import { CustomStyleHookMapping } from '../../utils/getStyleHookProps';
 
 /**
  * @internal
  */
-export function CompositeItem<Metadata>(componentProps: CompositeItem.Props<Metadata>) {
-  const { render, className, itemRef = null, metadata, ...elementProps } = componentProps;
+export function CompositeItem<Metadata, State extends Record<string, any>>(
+  componentProps: CompositeItem.Props<Metadata, State>,
+) {
+  const {
+    render,
+    className,
+    state = EMPTY_OBJECT as State,
+    props = EMPTY_ARRAY,
+    refs = EMPTY_ARRAY,
```

Comment:
It would be nice to standardize the API of the `refs` prop with `useRender` and `useRenderElement`: call it `ref`, and accept an array optionally.

Reply:
Since it's a component, it doesn't work properly (especially in React <19)